syntax = "proto3";

package proto_svg;

message Point {
	double x = 1;
	double y = 2;
}

message Rgb {
	int32 red = 1;
	int32 green = 2;
	int32 blue = 3;
}

message Rgba {
	int32 red = 1;
	int32 green = 2;
	int32 blue = 3;
	double opacity = 4;
}

message Color {
	string color = 1;
	Rgb rgb = 2;
	Rgba rgba = 3;
}
